### PR TITLE
Mint token motion component

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

This PR adds in dump MintTokenMotion component and types

**New stuff** ✨

* MintTokenMotion component
* colonyMotions enum
* Message descriptors for mint token motion

Screenshot 2021-03-19 at 10 48 17

Resolves DEV-254

### CODE REVIEW

File: src/i18n/en-motions.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { ColonyMotions } from '~types/index';
+
+const motionsMessageDescriptors = {
+  'motion.title': `{actionType, select,
+      ${ColonyMotions.MintTokensMotion} {Mint {amount} {tokenSymbol}}
+      other {Generic action we don't have information about}
```
Comment (rdig) [inline src/i18n/en-motions.ts:6]:
```suggestion
      other {Generic motion we don't have information about}
```

File: src/i18n/en-motions.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    }`,
+  'motion.type': `{actionType, select,
+      ${ColonyMotions.MintTokensMotion} {Mint Tokens}
+      other {Generic}
+    }`,
+};
```
Comment (rdig) [inline src/i18n/en-motions.ts:12]: These we might just end up re-using from actions, but leave it like this for now.

We'll decide when we actually get to use it.

File: src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+const displayName = 'dashboard.ActionsPage.MintTokenMotion';
+
+const MintTokenMotion = () => {
+  return <div />;
+};
+
+MintTokenMotion.displayName = displayName;
+
+export default MintTokenMotion;
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx:11]: This needs a bit more content:
- header title
- annotation hash
- details widget

Have a look at the `RecoveryAction`  component and copy them from there _(adapted to this motion, of course)_

File: src/types/colonyMotions.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export enum ColonyMotions {
+  /*
+    I dont like Motion postfitx, but if we would use actionType for
```
Comment (rdig) [inline src/types/colonyMotions.ts:3]: Motion postfitx ?

File: src/redux/actionTypes.ts [hunk 5]
```diff
@@ -79,6 +79,9 @@ export enum ActionTypes {
   COLONY_ACTION_UNLOCK_TOKEN = 'COLONY_ACTION_UNLOCK_TOKEN',
   COLONY_ACTION_UNLOCK_TOKEN_ERROR = 'COLONY_ACTION_UNLOCK_TOKEN_ERROR',
   COLONY_ACTION_UNLOCK_TOKEN_SUCCESS = 'COLONY_ACTION_UNLOCK_TOKEN_SUCCESS',
+  COLONY_MOTION_MINT_TOKENS = 'COLONY_MOTION_MINT_TOKENS',
+  COLONY_MOTION_MINT_TOKENS_ERROR = 'COLONY_MOTION_MINT_TOKENS_ERROR',
+  COLONY_MOTION_MINT_TOKENS_SUCCESS = 'COLONY_MOTION_MINT_TOKENS_SUCCESS',
```
Comment (rdig) [inline src/redux/actionTypes.ts:84]: Why did you add these here? They are not related to this PR.

Also, I think @chinins already added the proper ones in #2418

**Review by rdig — CHANGES_REQUESTED** [review]
Mostly good, but the actual component still needs more content

File: src/redux/actionTypes.ts (hunk 5 above)
Comment (chinins) [reply src/redux/actionTypes.ts:84]: I did indeed, @alicjakujawa you can cherry-pick my commits if you need to.

File: src/types/colonyMotions.ts (hunk 4 above)
Comment (alicjakujawa) [reply src/types/colonyMotions.ts:3]: "MintTokensMotion" so its like ColonyMotions.MintTokensMotion double motion word

**Review by rdig — APPROVED** [review]
Now it looks very nice!
